src/test/resources/base/dataclasses/base/BaseData.d.json

```diff
@@ -0,0 +1,15 @@
+{
+  "$schema" : "https://json-schema.axonivy.com/data-class/13.1.0/data-class.json",

```

Comment:
not sure whether we shouldn't better stick to the still supported 12.x versions

Reply:
maybe this would be better. but then we may better introduce excplicit tests for this. compileWithLastLTs()